.github/workflows/pull.yml

```diff
@@ -14,4 +14,4 @@ jobs:
         with:
           extraKey: pr-${GITHUB_HEAD_REF}
       - name: Check/Compile/Test
-        run: sbt check
+        run: sbt versionPolicyCheck check
```

Comment:
It would make more sense to put `versionPolicyCheck` into `check`

Reply:
I thought about that but this is supposed to be run on PRs only and we also run `check` during release where you want to run `versionCheck`. So I did it this way.